addons/sourcemod/scripting/scp_sf.sp

```diff
@@ -1386,16 +1386,19 @@ public Action OnDropItem(int client, const char[] command, int args)
 			if(entity > MaxClients)
 			{
 				WeaponEnum weapon;
-				bool big = (Items_GetWeaponByIndex(GetEntProp(entity, Prop_Send, "m_iItemDefinitionIndex"), weapon) && weapon.Type==ITEM_TYPE_WEAPON);
-
+				int index = GetEntProp(entity, Prop_Send, "m_iItemDefinitionIndex");
+				int dropType = GetEntityFlags(client) & FL_DUCKING ? ItemDrop_Drop : ItemDrop_Throw;
+				bool big = ((Items_GetWeaponByIndex(index, weapon) && weapon.Type==ITEM_TYPE_WEAPON) || index == ITEM_INDEX_MICROHID);
```

Comment:
I wonder if we should just make micro be a weapon type, I believe only change this would do is max amount of weapons able to be carried, which makes more sense to have it so?

Reply:
it does make more more sense, but I guess there's some fun in being able to pick it up with no restriction. I'm sure this was thought about at some point.
there are some other places in the code where this separate weapon type/micro check is done, so if it was to be done, maybe it is a little task for a different day.